### DISCUSSION TITLE
Outlook: Reversal of CC adjustments to AZ defaults

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#4203] [General]
Besides the initial idea to fix AzerothCore, Chromiecraft implemented a set of changes and adjustments to replicate the original progression to the respective (currently vanilla) progression. Some customizing are likely to stay (portals in the cities, etc.), but there is a set of changes which are suspected to be rolled back at any given point in time.

Unfortunately, nothing has been communicated and documented so far, thus here a set of questions that are hopefully answered in due time for transparencies sake:

1. Raid Size limited to 25. Blizzlike, raids like MC, BWL and future AQ40 are supposed to be accessible with up to 40 players. Initially, MC was released as a 20 players raid. Are there plans to roll back those changes and at which point in time of the progression for each of those 3 dungeons.
2. Onyxia was added as additional content, e.g. for the Shifting Sands quest and for access to her head and the respective quest rewards. She is an relevant part of the Shifting Sands quest line, but also influences the loot table of Nefarian in BWL. Is there any timeline when Onyxia is made inaccessible again, given time to verify the blizzlike (in the sense of AzerothCore blizzlike) loot tables for Nefarian and functionality of the questchain collecting the Draconic dummies through alternative means?
   - Given the retail progression, it would make sense to change this only back once we leave TBC from a progression point of view. This would delay the validation period for a longer time tho with respect to Nefarians loot table.
 3. Raid tuning: The raid tuning was mostly introduced to allow for some challenge / compensate for the increased damage and stats in WOTLK compared to Vanilla. Some of those tunings have been reduced gradually already (UBRS, and indirectly MC through the increase in raid group size from 20 to 25 players)
    - What are the expected timelines for MC, BWL, up to and including AQ40 to be returned to a state of no additional adjustments given future progression? (e.g. all but AQ40 are returned to AzerothCore defaults one we start the first TBC bracket and AQ40 is being set to defaults once we reach the level 70 bracket)
 4. Blizzard adjusted the costs of "current" pvp gear and the requirements once an expansions pre-patch was released, but before the actually expansion started. Can we expect a similar procedure here, giving access to level 60 PvP gear for their default Wotlk costs once the first TBC bracket starts?
 5. Worldbosses are currently spawning in a fixed position and phase the raid group. This is a custom change and is on the one hand to be expected to be rolled back in the future, but also to be applied for future worldbosses - based on the current considerations.

I am sure there are more relevant questions, but from my point of view it would be nice to have some more transparency what adjustments we can expect related to which part of the progression (not a when in the sense of time, but a when in sense of content)

**Comment by 55Honey** [comment]
Hi @balleny,
Thanks for being interested in our progression. 
I'd like to point you towards the #announcements channel on our discord and the news-section on ChromieCraft.com

In general, you can expect us to follow the blizzlike route when it comes to the points you've mentioned.

Once the details of any of these are meant to be published, you will find them in the channels above 

Regards,
Honey
